Content.Client/Clickable/ClickableComponent.cs

```diff
@@ -10,6 +10,7 @@ namespace Content.Client.Clickable
     public sealed partial class ClickableComponent : Component
     {
         [Dependency] private readonly IClickMapManager _clickMapManager = default!;
+        [Dependency] private readonly EntityManager _entMan = default!;
```

Comment:
This needs to be ECS'd eventually, but this will do for now.

Reply:
Would be better to leave as is so it gets some visibility that it needs fixing.

Reply:
Reverted this file